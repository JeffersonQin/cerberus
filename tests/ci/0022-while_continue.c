int main(void) {
  int ret = 0;
  int n = 11;
  while (n > 0) {
    n--;
    if (n % 2 == 0)
      continue;
    ret = ret + n;
    
  }
  return ret; // 25
}
