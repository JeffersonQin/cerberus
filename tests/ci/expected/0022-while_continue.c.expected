Defined {value: "Specified(25)", stdout: "", blocked: "false"}
CONSTRS ==> []
Log[0]

End[0]
